src/components/cluster/Cluster.tsx

```diff
@@ -59,9 +59,16 @@ export default class ClusterList extends Component<ClusterListProps, any> {
         this.initialise();
     }
 
+    componentDidUpdate(prevProps) {
```

Comment:
we could have used useffect?

Reply:
This is class component, useEffect hook is for functional component